web/src/features/auth/slices/authenticationSlice.ts

```diff
@@ -67,6 +71,20 @@ const {
 
 export default authSlice.reducer
 
+export const decodeRoles = (token: string | undefined) => {
+  if (isUndefined(token)) {
+    return []
+  }
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const decodedToken: any = jwt_decode(token)
+  try {
+    return decodedToken.resource_access['wps-web'].roles
```

Comment:
This should probably come from config. There's a variable REACT_APP_KEYCLOAK_CLIENT - that gets pulled in along with the auth url and the realm.